internal/server/statvar/golden/get_statvar_summary/unemployment.json

```diff
@@ -197,62 +179,57 @@
               },
               "earliestDate": "1976-01",
               "latestDate": "2021-12",
-              "numObservations": "2367090",
               "observationCount": 2367090,
-              "numTimeSeries": "6327",
               "timeSeriesCount": 6327,
               "placeTypeSummary": {
                 "City": {
-                  "numPlaces": "3055",
                   "placeCount": 3055,
                   "topPlaces": [
                     {
-                      "dcid": "geoId/3651000",
-                      "name": "New York"
+                      "dcid": "geoId/5645050",
+                      "name": "Laramie"
                     },
                     {
-                      "dcid": "geoId/0644000",
-                      "name": "Los Angeles"
+                      "dcid": "geoId/5631855",
+                      "name": "Gillette"
                     },
                     {
-                      "dcid": "geoId/1714000",
-                      "name": "Chicago"
+                      "dcid": "geoId/5613900",
+                      "name": "Cheyenne"
                     }
                   ]
                 },
                 "County": {
-                  "numPlaces": "3220",
                   "placeCount": 3220,
                   "topPlaces": [
                     {
-                      "dcid": "geoId/06037",
-                      "name": "Los Angeles County"
+                      "dcid": "geoId/72153",
```

Comment:
This diff is something to understand better.  What is the merging preference?

Currently, we pick the top-populated places.  Somehow, these are not that... it could be that we may need to use population-side-table here too....

Reply:
Place type summary field is to choose the table with most places. In this case, branch and base table have same number of places and branch cache data is used. Should it include the population side table?